third_party/bigframes_vendored/pandas/core/arrays/arrow/accessors.py

```diff
@@ -6,6 +6,79 @@
 from bigframes import constants
 
 
+class ListAccessor:
+    """
+    Accessor object for list data properties of the Series values.
+    """
+
+    def len(self):
+        """
```

Comment:
It looks like the doc format could be aligned with Google's style. Could you take another look at the formatting in `StringMethod.str()`?

Reply:
Good call. I was not aware that the formats are different. Updated.